bucket/julia.json

```diff
@@ -13,22 +13,11 @@
         }
     },
     "installer": {
-        "file": "julia-installer.exe",
-        "args": [
-            "/S",
-            "/D=$dir"
+        "script": [
+            "Expand-7zipArchive \"$dir\\julia-installer.exe\" \"$dir\" -Removal",
+            "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall.exe\" -Recurse -Force"
         ]
     },
-    "uninstaller": {
-        "file": "uninstall.exe",
-        "args": "/S"
-    },
-    "env_set": {
```

Comment:
Why to remove these sets?

Reply:
I've installed julia using original installer, and it didn't set these environment variables. These may be set by previous version, and didn't be used by current one.